packages/fiber/src/core/hooks.ts

```diff
@@ -89,10 +89,10 @@ export function useLoader<T, U extends string | string[]>(
 
 useLoader.preload = function <T, U extends string | string[]>(Proto: new () => LoaderResult<T>, input: U) {
   const keys = (Array.isArray(input) ? input : [input]) as string[]
-  return preload(Proto, ...keys)
+  return (preload as any)(Proto, ...keys)
```

Comment:
i just notice, these all have a slightly different syntax in suspend-react: fn([...keys]). use-asset had kind of a mixed api where some would call for an array and some for rest args. never could remember which is which. SR only uses array notation.

Reply:
ps that goes for clear, preload and peek.

Reply:
Awesome, I reverted the change and wrapped keys in arrays.